gen/KubernetesGenerator/templates/Kubernetes.cs.template

```diff
@@ -310,6 +309,39 @@ namespace k8s
             CustomInitialize();
         }
 
+        private async Task<HttpOperationResponse<T>> CreateResultAsync<T>(HttpRequestMessage httpRequest, HttpResponseMessage httpResponse, bool? watch, CancellationToken cancellationToken)
```

Comment:
Creating this method reduced the size of KubernetesClient.dll by about 50K (1 percent)